dotcom-rendering/scripts/webpack/frontend.js

```diff
@@ -22,16 +23,20 @@ const commonConfigs = ({ platform }) => ({
 			: 'eval-cheap-module-source-map',
 	resolve: {
 		alias: {
-			'@root': path.resolve(__dirname, '.'),
-			'@frontend': path.resolve(__dirname, 'src'),
+			// '@root': path.resolve(__dirname, '.'),
```

Comment:
Esbuild was struggling to resolve these aliases.  The `TsconfigPathsPlugin` used below resolves this issue in the majority of cases.